apps/projects/app/components/Panel/NewIssue/NewIssue.js

```diff
@@ -5,18 +5,26 @@ import { Mutation } from 'react-apollo'
 import { gql } from 'apollo-boost'
 import { theme, Field, Info, TextInput, Button, DropDown } from '@aragon/ui'
 import { NEW_ISSUE, GET_ISSUES } from '../../../utils/gql-queries.js'
+import { Form } from '../../Form'
+import ethereumLoadingAnimation from '../../../assets/svg/ethereum-loading.svg'
 
 // TODO: labels
 // TODO: import validator from '../data/validation'
 
-const Form = styled.form`
-  display: grid;
-  grid-template-columns: 1fr;
-  column-gap: 20px;
-  > :last-child {
-    margin-top: 20px;
-  }
-`
+const Creating = () => (
+  <div
+    style={{
+      display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'center',
+      height: '100%',
+      flexDirection: 'column',
+    }}
+  >
+    <LoadingAnimation src={ethereumLoadingAnimation} />
```

Comment:
This asset isn't rendering for me, it's just showing as a broken link.

Reply:
Fixed, assets cleaned up.